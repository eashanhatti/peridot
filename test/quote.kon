let {
  assume Rel : [_ : (Quote qUnit)] -> MType;
} in {
  (pure unit)
}
