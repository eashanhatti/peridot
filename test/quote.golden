((fromList [(Global {unGlobal = 0}, Just UnitType)],
  fromList [],
  fromList [],
  []),
 Let
   [MetaConstant
      (Id {unId = 3})
      (MetaFunType
         Explicit
         (QuoteType (QuoteIntro UnitType))
         (TypeType Meta))]
   (IOIntroPure UnitIntro))
